app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenActionCard.kt

```diff
@@ -44,7 +44,7 @@ fun BitwardenActionCard(
     cardTitle: String,
     actionText: String,
     onActionClick: () -> Unit,
-    onDismissClick: () -> Unit,
+    onDismissClick: (() -> Unit)? = null,
```

Comment:
Can we move this below the modifier, the pattern is that the modifier is always the first param with a default value